full/coupler_main.F90

```diff
@@ -562,47 +562,25 @@ program coupler_main
           call coupler_flux_down_from_atmos(Atm, Land, Ice, Land_ice_atmos_boundary, Atmos_land_boundary, &
               Atmos_ice_boundary, Time_atmos, current_timestep, coupler_clocks, coupler_chksum_obj)
 
-          !      --------------------------------------------------------------
-          !      ---- land model ----
-          call fms_mpp_clock_begin(coupler_clocks%update_land_model_fast)
-          if (do_land .AND. land%pe) then
-            if (land_npes .NE. atmos_npes) call fms_mpp_set_current_pelist(Land%pelist)
-            call update_land_model_fast( Atmos_land_boundary, Land )
-          endif
-          if (land_npes .NE. atmos_npes) call fms_mpp_set_current_pelist(Atm%pelist)
```

Comment:
Since this is now done within the added routines, if the`do_ice/do_land`conditionals are false this call won't be made. I don't know if we really need that call anyway in that case but wanted to point it out.

Reply:
You're right, and we've let the scientists know CHECKSUMS won't be computed if the components do not change